src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1217,7 +1220,14 @@ public static VariantContext simpleMerge(final Collection<VariantContext> unsort
 
         final String ID = rsIDs.isEmpty() ? VCFConstants.EMPTY_ID_FIELD : Utils.join(",", rsIDs);
 
-        final VariantContextBuilder builder = new VariantContextBuilder().source(name).id(ID);
+        // This preserves the GATK3-like behavior of reporting multiple sources, delimited with hyphen:
+        final String allSources = variantSources.isEmpty() ? name : variantSources.stream()
+                .sorted()
```

Comment:
I am concerned about doing a sort on the source names for every single merged VCF record we're writing -- with thousands or tens of thousands of samples, that would get expensive!

Reply:
That's reasonable - what about what I proposed a little earlier in this thread and overloading the method to make the existing default case unchanged (list the first source), but have the ability for code to opt-in to the behavior of tracking all sources? I am happy to write that.

Reply:
@bbimber That option is looking more attractive -- if you'd like to implement it, please go ahead and tag us for review. If we don't touch the default behavior this will be much easier to get in!